src/main/kotlin/org/rust/ide/intentions/ToggleFeatureIntention.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.intentions
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.util.parentOfType
+import org.rust.RsBundle
+import org.rust.cargo.project.model.cargoProjects
+import org.rust.cargo.project.workspace.PackageOrigin
+import org.rust.lang.core.RsPsiPattern.insideAnyCfgFeature
+import org.rust.lang.core.psi.ext.RsElement
+import org.rust.lang.core.psi.ext.cargoProject
+import org.rust.lang.core.psi.ext.containingCargoPackage
+
+class ToggleFeatureIntention : RsElementBaseIntentionAction<ToggleFeatureIntention.Context>() {
+    private val matcher = insideAnyCfgFeature
+
+    data class Context(val featureName: String, val element: RsElement)
+
+    override fun getFamilyName() = RsBundle.message("intention.Rust.ToggleFeatureIntention.family.name")
+
+    override fun findApplicableContext(project: Project, editor: Editor, element: PsiElement): Context? {
+        if (!matcher.accepts(element)) return null
+
+        val context = element.parentOfType<RsElement>() ?: return null
+        val featureName = element.text.trim { it == '"' }
```

Comment:
```suggestion
        val context = element.parentOfType<RsLitExpr>() ?: return null
        val featureName = element.stringValue
```

Reply:
This wouldn't work for unquoted features, right?
```rust
#[cfg(feature = foo/*caret*/)]
fn foo() {}
```
`foo` here isn't a `RsLitExpr`.

Reply:
Hmm, feature names without quotes are not valid? :laughing: I wonder where I saw it, I remember specifically wanting to support it, I even created specific tests for it. Damn.

Reply:
Oh, right, there's a test for it in `RsPsiPatternTest.kt` (`test in cfg feature without literal`). I didn't actually check if it's valid syntax :laughing:

Reply:
The pattern supports a feature without quotes for the completion case - if a user invokes the completion right after `=` token